src/main/groovy/pl/allegro/tech/build/axion/release/VerifyReleaseTask.groovy

```diff
@@ -25,7 +29,7 @@ class VerifyReleaseTask extends DefaultTask {
 
         if (resolver.checkUncommittedChanges()) {
             boolean uncommittedChanges = repository.checkUncommittedChanges()
-            project.logger.quiet("Looking for uncommitted changes.. ${uncommittedChanges ? 'FAILED' : ''}")
+            logger.trace("Looking for uncommitted changes.. ${uncommittedChanges ? 'FAILED' : ''}")
```

Comment:
QUIET log level in gradle (despite its name) is higher than WARNING - it's always visible. You cannot map it to trace level which, I bet, nobody will see ever.